dockstore-webservice/src/test/java/io/dockstore/webservice/permissions/sam/SamPermissionsImplTest.java

```diff
@@ -282,6 +290,41 @@ public void userNotInSamReturnsEmptyMap() throws ApiException {
         Assert.assertEquals(0, sharedWithUser.size());
     }
 
+    /**
+     * Test that a reader gets herself back
+     * @throws ApiException
+     */
+    @Test
+    public void getPermissions1() throws ApiException {
```

Comment:
`testSelfPermissions` would be more descriptive

Reply:
👍